.prettierrc

```diff
@@ -4,7 +4,7 @@
 	"trailingComma": "es5",
 	"overrides": [
 		{
-			"files": "*.md",
+			"files": ["*.md", "*.json"],
```

Comment:
This will conflict with npm formatting.

Reply:
Hm so maybe it’s VSCode `format on save` doing its thing but if I remove *.json from `prettierrc`overrides, then I can’t have `package.json` files rendered with spaces instead of tabs as I think you requested?

Should I replace `*.json` with `package.json`?

Reply:
Nope, we shouldn't format package.json with Prettier, because npm will reformat it on install. Looks like you need to tweak your editor setup ;-/

Reply:
Is it ok if I add a `.prettierignore` file then? I think a lot of people have `formatOnSave` set to `true`... See this:
https://medium.com/@martin_hotell/stop-re-formatting-package-json-with-prettier-and-vscode-once-and-for-all-52d283067f9a

Reply:
I have it scoped for JS:

```json
  "[javascript]": {
    "editor.formatOnSave": true
  },
  "[javascriptreact]": {
    "editor.formatOnSave": true
  },
```

But feel free to add it to .prettierignore too.

Reply:
Done in 50e4fd9f9af327d0c96afb73dcdf83114b160877